styles/_variables.scss

```diff
@@ -8,3 +8,4 @@ $half-whitespace-unit: $standard-whitespace-unit / 2;
 
 /* layout */
 $standard-gutter-width: 10px;
+$standard-footer-breakpoint: 599px;
```

Comment:
That's the value that is used in the login page. The style guide has no breakpoints builtin (they are all set in workwise). So should we also use $breakpoint-medium in the style guide or maybe move the responsive part to workwise?